guides/source/developers/getting-started/develop-solidus.html.md

```diff
@@ -102,7 +102,7 @@ DB=postgresql bundle exec rspec
 
 ### Generate a code coverage report
 
-You can generate a [SimpleCov](https://github.com/colszowka/simplecov) code
+You can generate a [SimpleCov](simplecov)) code
```

Comment:
I believe the syntax you are intending to use here is `[SimpleCov][simplecov]`.

See reference-style links in the [Markdown spec](https://daringfireball.net/projects/markdown/syntax#link).

Reply:
@benjaminwil Thanks noticed this just a few seconds ago 👍